app/models/event.rb

```diff
@@ -10,4 +11,8 @@ class Event < ApplicationRecord
             presence: true
 
   scope :upcoming, -> { where('date >= ?', Time.zone.today) }
+  scope :requiring_notification, -> do
+    where('date = ?', Date.current - NOTIFICATION_PERIOD)
+      .includes(presentations: [:person])
```

Comment:
Alright got this one figured out. The scope in the event model is just for itself and in the event job I have...
```
class EventNotificationJob < ApplicationJob
  def self.perform(event)
    event.presentations.joins(:person).each do |presentation|
      PersonMailer.presentation_notification(
        presentation.person,
        presentation.event.date
      ).deliver
    end
  end
end
```